rllib/core/rl_module/marl_module.py

```diff
@@ -403,12 +400,24 @@ class MultiAgentRLModuleSpec:
         module_specs: The module specs for each individual module. It can be either a
             SingleAgentRLModuleSpec used for all module_ids or a dictionary mapping
             from module IDs to SingleAgentRLModuleSpecs for each individual module.
+        load_state_path: The path to the module state to load from. NOTE: This must be
```

Comment:
Nice. Thanks for preemptively answering my questions ;)

Reply:
dude @kouroshHakha, why is a path part of the spec?
would we save this path with the serialized module?

Reply:
No it's for when a user want to load up a new MARL module / part of a marl module from an old (already checkpointed) one via setting this attributed.

Reply:
that's sounds perfect. thanks for the explanation.